feature-libs/product-configurator/rulebased/components/attribute/types/checkbox-list/configurator-attribute-checkbox-list.component.ts

```diff
@@ -136,4 +138,41 @@ export class ConfiguratorAttributeCheckBoxListComponent
       this.onHandleAttributeQuantity(eventObject);
     }
   }
+
+  getAriaLabel(
+    value: Configurator.Value,
+    attribute: Configurator.Attribute
+  ): string {
+    value = this.mergePriceAndValue(value);
+    let params;
+    let key;
+    if (value.valuePriceTotal && value.valuePriceTotal?.value !== 0) {
+      key = 'configurator.a11y.valueOfAttributeFullWithPrice';
+      params = {
+        value: value.valueDisplay,
+        attribute: attribute.label,
+        price: value.valuePriceTotal.formattedValue,
+      };
+    } else if (value.valuePrice && value.valuePrice?.value !== 0) {
+      key = 'configurator.a11y.valueOfAttributeFullWithPrice';
+      params = {
+        value: value.valueDisplay,
+        attribute: attribute.label,
+        price: value.valuePrice.formattedValue,
+      };
+    } else {
+      key = 'configurator.a11y.valueOfAttributeFull';
+      params = {
+        value: value.valueDisplay,
+        attribute: attribute.label,
+      };
+    }
+    let ariaLabel = '';
+    this.translation
+      .translate(key, params)
+      .pipe(take(1))
+      .subscribe((text) => (ariaLabel = text));
+
+    return ariaLabel;
+  }
```

Comment:
@steinsebastian: maybe you can follow the same pattern for the method you adapted in your PR i commented earlier.

Reply:
done